react/features/lobby/functions.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import { getCurrentConference } from '../base/conference';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Returns a displayable name for the knocking participant.
+ *
+ * @param {string} name - The received name.
+ * @returns {string}
+ */
+export function getKnockingParticipantDisplayName(name: string) {
+    if (name) {
+        return name;
+    }
+
+    return 'Fellow Jitster';
```

Comment:
How is it possible that they know without a name? Shouldn't we auto-reject them in that case? At any rate, this should be taken from interfaceConfig, with a fallback.